crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -89,11 +89,12 @@ pub fn cli() -> clap::Command {
                 .value_parser(clap::value_parser!(Language))
                 .help("The language to generate"),
         )
-        .arg(Arg::new("delete_files").long("delete-files").action(SetTrue).help(
-            "Delete outdated generated files whose definitions have been \
-             removed from the module. Prompts before deleting unless --force is \
```

Comment:
I'll comment here as well that this help text is also super misleading. Files which existed from the previous generation will be overridden for the new generation (for example, when you update the schema of a table). This will completely overwrite that file including any modifications to it as long as we detect that our special file header is uneditied.